apps/api-server/seeds/development.js

```diff
@@ -107,6 +114,13 @@ module.exports = async function seed(config, db) {
           withExisting: 'error',
         }
       },
+      emailConfig: {
+        notifications: {
+          fromAddress: 'nobody@openstad.dev',
```

Comment:
Is nobody@openstad.dev nu ook een werkende default? Bij oud openstad als je namelijk een nieuwe site maakt en je voert een verkeerde from email in, dan werkt inloggen niet. Denk dat een werkende default wel belangrijk is